tests/test_TFUtil.py

```diff
@@ -143,6 +143,26 @@ def test_Data_unknown_feature_no_time():
   assert d1.batch_shape == (None, None)
 
 
+def test_Data_copy_with_feature_dim_axis():
+  import numpy as np
+  size_placeholder = tf.constant(np.full((10,), 10), dtype=tf.int32)
+  d = Data(name='test_data', shape=(None, 64, 64), dtype='float32',
```

Comment:
Do not use `(None, 64, 64)` but better different numbers, e.g. `(None, 13, 17)`.